specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -2704,6 +2898,46 @@
         "dataset"
       ]
     },
+    "KustoCommand": {
```

Comment:
Are we releasing this type? I thought Kusto team has another design for this. Please sync with Tomer first.

Reply:
I've already asked Tomer and he said that "KustoCommand" needs to be updated to public. I will sync with Tomer again.